x-pack/legacy/plugins/spaces/public/advanced_settings/advanced_settings_service.test.tsx

```diff
@@ -5,32 +5,34 @@
  */
 
 import { AdvancedSettingsService } from './advanced_settings_service';
-jest.mock('ui/management', () => {
-  return {
-    PAGE_TITLE_COMPONENT: 'page_title_component',
-    PAGE_SUBTITLE_COMPONENT: 'page_subtitle_component',
-  };
-});
 
 describe('Advanced Settings Service', () => {
   describe('#setup', () => {
     it('registers space-aware components to augment the advanced settings screen', () => {
       const deps = {
         getActiveSpace: jest.fn().mockResolvedValue({ id: 'foo', name: 'foo-space' }),
-        registerSettingsComponent: jest.fn(),
+        componentRegistry: {
+          register: jest.fn(),
+          get: jest.fn(),
+          componentType: {
+            PAGE_TITLE_COMPONENT: 'page_title_component',
+            PAGE_SUBTITLE_COMPONENT: 'page_subtitle_component',
+            PAGE_FOOTER_COMPONENT: 'page_footer_component',
+          },
```

Comment:
Maybe we should be providing these in a public `mocks.ts` as we have done in other plugins?